test/java/seedu/addressbook/testutil/TypicalTestPersons.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.testutil;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.UniquePersonList;
+
+/**
+ * Class to generate typical test persons
+ */
+public class TypicalTestPersons {
+
+    public static TestPerson claude, gates, jobs, donald, hopper;
+
+    public TypicalTestPersons() {
+        try {
+            claude = new PersonBuilder().withName("Claude Shannon").withPhone("91234567", false)
+                    .withEmail("cs@gmail.com", false).withAddress("1 Clementi Road", true).build();
+
+            gates = new PersonBuilder().withName("Bill Gates").withPhone("92434567", false)
+                    .withEmail("bg@gmail.com", false).withAddress("2 Clementi Road", false).build();
+
+            jobs = new PersonBuilder().withName("Steve Jobs").withPhone("91934267", true)
+                    .withEmail("sj@gmail.com", true).withAddress("3 Clementi Road", false).build();
+
+            donald = new PersonBuilder().withName("Donald Knuth").withPhone("91214567", true)
+                    .withEmail("dk@gmail.com", false).withAddress("4 Clementi Road", true).build();
+
+            hopper = new PersonBuilder().withName("Grace Hopper").withPhone("83434567", true)
+                    .withEmail("gh@gmail.com", true).withAddress("5 Clementi Road", true).build();
+
```

Comment:
When defining test data, try to follow a pattern that is easy to remember. For example, the first person's name has initials A. B.  (e.g. Alice Betsy), second person's name has initials B. C. (Ben Charles) and so on. When things always follow a pattern, anything going out of that pattern (e.g. Ben suddenly appears at the start of the list) gets noticed immediately and sometimes can lead to discover bugs.